src/dep/vs2010/pre.bat

```diff
@@ -1,44 +1,12 @@
 @echo off
-cd ..\..\src\dep\
 mkdir generated
-cd scripts
-echo %CD%
+cd ..\..\src\dep\scripts
 echo running perl scripts
-perl cave_sprite_func.pl -o ../generated/cave_sprite_func.h
-perl license2rtf.pl -o ../generated/license.rtf ../../license.txt
-perl cave_tile_func.pl -o ../generated/cave_tile_func.h
-perl neo_sprite_func.pl -o ../generated/neo_sprite_func.h
-perl psikyo_tile_func.pl -o ../generated/psikyo_tile_func.h
-perl toa_gp9001_func.pl -o ../generated/toa_gp9001_func.h
-perl gamelist.pl -o ../generated/driverlist.h -l ../../gamelist.txt ../../burn/drv/capcom ../../burn/drv/cave ../../burn/drv/cps3 ../../burn/drv/dataeast ../../burn/drv/galaxian ../../burn/drv/irem ../../burn/drv/konami ../../burn/drv/megadrive ../../burn/drv/neogeo ../../burn/drv/pce ../../burn/drv/pgm ../../burn/drv/pst90s ../../burn/drv/pre90s ../../burn/drv/psikyo ../../burn/drv/sega ../../burn/drv/taito ../../burn/drv/toaplan ../../burn/drv/snes ../../burn/drv/sms
+perl cave_sprite_func.pl -o ../../../projectfiles/visualstudio-2010/generated/cave_sprite_func.h
+perl license2rtf.pl -o ../../../projectfiles/visualstudio-2010/generated/license.rtf ../../license.txt
+perl cave_tile_func.pl -o ../../../projectfiles/visualstudio-2010/generated/cave_tile_func.h
+perl neo_sprite_func.pl -o ../../../projectfiles/visualstudio-2010/generated/neo_sprite_func.h
+perl psikyo_tile_func.pl -o ../../../projectfiles/visualstudio-2010/generated/psikyo_tile_func.h
+perl toa_gp9001_func.pl -o ../../../projectfiles/visualstudio-2010/generated/toa_gp9001_func.h
+perl gamelist.pl -o ../../../projectfiles/visualstudio-2010/generated/driverlist.h -l ../../../projectfiles/visualstudio-2010/generated/gamelist.txt ../../burn/drv/atari ../../burn/drv/capcom ../../burn/drv/cave ../../burn/drv/coleco ../../burn/drv/cps3 ../../burn/drv/dataeast ../../burn/drv/galaxian ../../burn/drv/irem ../../burn/drv/konami ../../burn/drv/megadrive ../../burn/drv/midway ../../burn/drv/msx ../../burn/drv/neogeo ../../burn/drv/pce ../../burn/drv/pgm ../../burn/drv/pst90s ../../burn/drv/pre90s ../../burn/drv/psikyo ../../burn/drv/sega ../../burn/drv/sg1000 ../../burn/drv/spectrum ../../burn/drv/taito ../../burn/drv/toaplan ../../burn/drv/snes ../../burn/drv/sms
 

```

Comment:
just curious: are those hardcoded paths going to cause a problem for anyone?  Barry?

Reply:
Barry may say differently but the paths are all relative from the .bat file so should be ok. I purposely .gitingore on the "generated" folder so it doesn't poo old headers in the repository. 

Also, I doubt he's using this .bat file as it seems to only be used in the IDE project files, but again I could be wrong.